src/game/Commands/AccountCommands.cpp

```diff
@@ -553,7 +553,7 @@ bool ChatHandler::HandleBanAllIPCommand(char* args)
     } while (result->NextRow());
 
     delete result;
-    if (result = CharacterDatabase.PQuery("SELECT `account` FROM `characters` WHERE `account` IN '%s' AND `level` > '%u' GROUP BY `account`", allAccounts.str().c_str(), maxLevel))
+    if (result = CharacterDatabase.PQuery("SELECT `account` FROM `characters` WHERE `account` IN (`%s`) AND `level` > '%u' GROUP BY `account`", allAccounts.str().c_str(), maxLevel))
```

Comment:
Bad change, you want '%s'.

Reply:
fixed with https://github.com/vmangos/core/pull/896/commits/4b5b0bf7ff8c5f6fd7abea694d2105fb8ddb9c8c